### DISCUSSION TITLE
How to transform the link into a link in code block

### DISCUSSION THREAD

**Discussion by changk99** [remarkjs/remark#1169] [Q&A] [answered]
### link
      [nuxt](https://content.nuxtjs.org/api/configuration/#anchorlinks)

I expect the (https://content.nuxtjs.org/api/configuration/#anchorlinks) can transform a link in code block,. How to configure remark to do that?

**Comment by wooorm** [comment]
I don’t understand what you mean. Please follow our support guide: https://github.com/remarkjs/.github/blob/main/support.md. Provide much, much more info.

**Reply by changk99** [reply]
In markdown file

```md
[nuxt](https://content.nuxtjs.org/1)

    [nuxt2](https://content.nuxtjs.org/2)
```
the https://content.nuxtjs.org/1 can transform a link, but https://content.nuxtjs.org/2 can't. I want to the https://content.nuxtjs.org/2 can be transform a link like https://content.nuxtjs.org/1

**Reply by wooorm** [reply]
I think you mean that markdown links (`[a](b)`) are transformed, but that things that *look* like markdown links inside code are not transformed?

Well, that’s for the same reason as that `function x() {}` is a JavaScript function and `"function y() {}"` is a JavaScript string: that’s the point.

That being said, you can do everything with remark. But you need to write your own plugins to do things (remark doesn‘t do thins by default).

If you have ideas for how Nuxt should work, then you should post that feature request to the Nuxt folks.

**Reply by changk99** [reply]
yes, I want the ```[a](b)```, if b is a link can transform a link.
in markdown-it
```js
md = new MarkdownIt({
  html: false,
  xhtmlOut: false,
  breaks: false,
  langPrefix: 'language-',
  linkify: false,
  typographer: false,
})
```
if  linkify is true then the b can be transfrom a link. so I thought remark had a similar configuration

**Reply by wooorm** [reply, marked as answer]
No, remark itself does not have options like this. https://github.com/remarkjs/remark#intro

**Reply by changk99** [reply]
Whether there is any remark plugin of this function?

**Reply by wooorm** [reply]
markdown-it does not do what you are asking here: https://markdown-it.github.io/#md3=%7B%22source%22%3A%22%5Ba%5D%28%23%29%5Cn%5Cn%20%20%20%20%5Ba%5D%28%23%29%22%2C%22defaults%22%3A%7B%22html%22%3Afalse%2C%22xhtmlOut%22%3Afalse%2C%22breaks%22%3Afalse%2C%22langPrefix%22%3A%22language-%22%2C%22linkify%22%3Atrue%2C%22typographer%22%3Atrue%2C%22_highlight%22%3Atrue%2C%22_strict%22%3Afalse%2C%22_view%22%3A%22html%22%7D%7D.

if you want to make a plugin that looks inside code blocks, you might want to use https://github.com/syntax-tree/hast-util-find-and-replace.

**Reply by changk99** [reply]
Thanks,I try to use in nuxt
